public/locales/fr/status.json

```diff
@@ -114,8 +115,12 @@
     "cannot-process": "Nous avons examiné votre candidature et ne pouvons plus la traiter.",
     "explanation": "Nous vous envoyons une lettre expliquant pourquoi, ainsi que vos pièces justificatives. Vous devriez les recevoir dans les prochaines semaines."
   },
-  "status-check-numerous-attempts": {
-    "description": "Si après de nombreuses tentatives vous ne parvenez pas à accéder au statut de votre candidature en ligne, ou si votre candidature prend plus de temps que prévu ",
+  "status-check-contact": {
+    "description-no-record": "Si après de nombreuses tentatives vous ne parvenez pas à accéder au statut de votre candidature en ligne, ou si votre candidature prend plus de temps que prévu ",
```

Comment:
Change "si votre candidature prend plus de temps que prévu " to "si votre candidature prend plus de temps que la "